ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/ip_name_server_groups.yml

```diff
@@ -0,0 +1,20 @@
+---
+ip_name_server_groups:
```

Comment:
Please check the coverage and improve it

Reply:
Coverage 98 percent(2 partial).